python/aibrix_kvcache/integration/vllm/vllm_v0.8.5-aibrix-kvcache.patch

```diff
@@ -5371,31 +5371,6 @@ index 33c4c3219..fe343b5ca 100644
  opentelemetry-exporter-otlp>=1.26.0,<1.27.0  # vllm.tracing
  opentelemetry-semantic-conventions-ai>=0.4.1,<0.5.0  # vllm.tracing
 +aibrix-kvcache # Required for kv cache offloading
```

Comment:
BTW, we need to test the release pipeline, currently, `aibrix-kvcache` is probably not managed by the same KEY.